diff --git a/Sources/XKit/Model/Entitlements/EntitlementTypes.swift b/Sources/XKit/Model/Entitlements/EntitlementTypes.swift
--- a/Sources/XKit/Model/Entitlements/EntitlementTypes.swift
+++ b/Sources/XKit/Model/Entitlements/EntitlementTypes.swift
@@ -28,6 +28,7 @@ extension EntitlementContainer {
         WirelessAccessoryEntitlement.self,
         HomeKitEntitlement.self,
         HealthKitEntitlement.self,
+        HealthKitBackgroundDeliveryEntitlement.self,
     ]
 }
 
@@ -168,3 +169,10 @@ public struct HealthKitEntitlement: Entitlement, RawRepresentable {
     public var rawValue: Bool
     public init(rawValue: Bool) { self.rawValue = rawValue }
 }
+
+public struct HealthKitBackgroundDeliveryEntitlement: Entitlement, RawRepresentable {
+    public static let identifier = "com.apple.developer.healthkit.background-delivery"
+
+    public var rawValue: Bool
+    public init(rawValue: Bool) { self.rawValue = rawValue }
+}
diff --git a/Sources/XKit/DeveloperServices/App IDs/Entitlements/DeveloperServicesCapability.swift b/Sources/XKit/DeveloperServices/App IDs/Entitlements/DeveloperServicesCapability.swift
--- a/Sources/XKit/DeveloperServices/App IDs/Entitlements/DeveloperServicesCapability.swift
+++ b/Sources/XKit/DeveloperServices/App IDs/Entitlements/DeveloperServicesCapability.swift
@@ -91,6 +91,12 @@ extension HealthKitEntitlement: EntitlementWithCapability {
     }
 }
 
+extension HealthKitBackgroundDeliveryEntitlement: EntitlementWithCapability {
+    var capability: DeveloperServicesCapability {
+        DeveloperServicesCapability(.healthkit, isFree: true)
+    }
+}
+
 extension HomeKitEntitlement: EntitlementWithCapability {
     var capability: DeveloperServicesCapability {
         DeveloperServicesCapability(.homekit, isFree: true)
